### ISSUE TITLE
Dumb glitch with linking account

### ISSUE THREAD

**Issue by Shadow-Mage** [chtaylo2/Roku-GooglePhotos#27]
I only recently learned how Google Photos works. I attempted to upload a large batch of pictures from Drive to Photos *in order of the date they were modified*, and it didn't work. Long story short, I tried unlinking and relinking my account on the Photos channel. Eventually I got things set up the way I wanted them, but now the stupid channel is forcing me to relink my account EVERY SINGLE TIME I OPEN THE CHANNEL. And every single time I do, I get an error message saying something like "0Auth0 no response received - [Try again | Back]. If I press the back button, it lets me into my account's Google Photos anyway. If I press Try Again and enter a new linking code, I'm given a notice saying my account is already linked and I get redirected to my Google Photos. But the minute I exit the channel, it seems to reset and I have to do the whole damn process over again the next time I open the channel. I've tried removing and reinstalling the channel, and it doesn't fix anything.

TL;DR Google Photos keeps making me relink my account every time I open the stupid channel. PLS FIX.

**Comment by chtaylo2** [comment]
Sounds like you may have a rogue registration in there some how... If you completely remove the channel, restart your Roku and reinstall, all the prior registry entries would be deleted. I know you said you did this, but that's the recommendation I would have to fix. 

Do you have any special characters in your name, email (besides @ or . ) ?  If you can take a video showing this and upload it somewhere i can retrieve I'll take a look but I've not run into anyone else stating the same thing, so have no other way to duplicate.

**Comment by chtaylo2** [comment]
I have a potential fix for the issue which I'd like to have you test, since I can't duplicate here. It'll have you install a private channel ID. Please give me your email address, and I'll send you the code.

**Comment by Shadow-Mage** [comment]
The only place I have special characters in my account data is my account password. It includes special characters for security, but it's never been needed for the channel.

I'm using a different account than my main one for photos. I have several alt accounts for each of my devices. I'll give you the email addresses for both my main account and the alt I'm using, in case that makes a difference.

My main account for email is:
ignatian.mystic@gmail.com

The account I've been using on the Google Photos channel is:
from.the.rca.viking.pro.volume3@gmail.com

I never bothered trying to restart my TV before reinstalling the channel. I'll try the remove/restart/reinstall method you suggested and let you know how it goes.

**Comment by Shadow-Mage** [comment]
Ugh, no dice. I tried removing the channel and restarting the TV before adding it again, but it didn't work. This is the dialog box I get on the TV channel every time I enter the activation code on my phone:

|            OAuth2 polling request failed         |
|                                                                         |
|   Missing User Data                                     |
|                                                                         |
|                                                 Try Again  []   |
|                                                          Back []   |

If you've got any other suggestions, I'd be happy to hear them. Otherwise, I think sending me that private channel ID you mentioned would be the best course of action for now.

**Comment by Shadow-Mage** [comment]
Darn GitHub, it messed up the recreation of the dialog box that I posted. The text in it is still the same, as are the options it gives me.

**Comment by chtaylo2** [comment]
Confirmed issue was related to a third party email. Google API does not provide a name when this type of account is used. Fixed in v.3.727.20170123.

Will be published to Roku store in the next couple weeks as part of Release v.1.6

**Comment by chtaylo2** [comment]
Could you please confirm if the test channel I provided fixed your issue?
